lib/io/fileutils/access_linux_test.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
+// +build !windows !darwin
```

Comment:
i'm puzzled; I thought the same rule applies to test or non-test files. Why did you remove the directive in access_linux.go but kept it here in access_linux_test.go?

Reply:
An oversight...